### DISCUSSION TITLE
Jellyfin插件能否切换到官方镜像

### DISCUSSION THREAD

**Discussion by diave971** [istoreos/istoreos#1625] [General]
新版本的官方jellfin已经支持了rk35xx系列的硬件解码，
istore里面的jellyfin插件能否提供官方的镜像支持呢，
最新版本的Windows端jellyfin media已无法使用老版本的jellfin server。
![屏幕截图 2024-06-07 021807](https://github.com/istoreos/istoreos/assets/60306114/f0362c53-6f1b-449a-a32e-4f14a4b39023)

**Comment by w9315273** [comment]
### [jellyfin docker 镜像](https://hub.docker.com/r/linuxserver/jellyfin)
反正 istore里面的jellyfin插件 也是docker
为什么不直接用docekr呢

**Reply by diave971** [reply]
arm机器的硬解自己配置的话还是挺麻烦的，插件点一下就装好了

**Reply by DaMccRee** [reply]
> arm机器的硬解自己配置的话还是挺麻烦的，插件点一下就装好了

转码功能用的上吗

**Reply by diave971** [reply]
> > arm机器的硬解自己配置的话还是挺麻烦的，插件点一下就装好了
> 
> 转码功能用的上吗

web播放还是挺不错的，就是rk3568支持常见的编码，对资源还是有点要求的

**Comment by jjm2473** [comment]
之前我测试过官方镜像好像不是很稳定，或者说文件兼容性比较差，有些文件播放几秒钟就挂掉了。如果你要测试jellyfin官方的镜像，可以执行这个命令配置：
```bash
uci set "jellyfin.@jellyfin[0].image=jellyfin/jellyfin"
uci commit jellyfin
```
然后在插件那边点重新部署就行了。部署以后需要在jellyfin的后台页面配置硬件加速。

不管官方镜像测试结果怎么样，都反馈下。

PS：如果要改回默认的镜像，执行：
```bash
uci set "jellyfin.@jellyfin[0].image=default"
uci commit jellyfin
```
部署以后需要在jellyfin的后台页面关掉硬件加速（如果测试官方镜像时打开了）。

**Reply by diave971** [reply]
测试了一下，修改后用插件部署容器，貌似勾选硬件加速cpu也会占用爆表，jellyfin ffmpeg进程占用cpu100%，虽然能顺利播放，这应该是没有正确调用硬解？
测试平台是r5c rk3568，
下面是我勾选的的硬件加速配置
![image](https://github.com/istoreos/istoreos/assets/60306114/694994c7-9fd5-4f31-a9c4-aa5f782fcc60)

**Reply by jjm2473** [reply]
cpu可能是被音频转码占用的，你可以试试播放双声道以内的文件。还有，播放时要打开信息窗口，看看是不是在转码。

**Reply by RonanYang** [reply]
更换镜像为：nyanmisaka/jellyfin:latest-rockchip
播放文件：WEB-DL.DDP.5.1.Atmos.H.265
测试平台：R6C rk3588s
硬件解码没问题

jellyfin-mpp我测试转码更改分辨率未生效

两个镜像测试的不充分，算是提供一个信息

**Comment by jjm2473** [comment]
jellyfin插件已经支持选择镜像。使用不同镜像测试时，最好使用不同的配置文件路径，以免配置错误影响测试效果
